app/src/main/java/com/communistutopia/spacetrader/viewmodel/DebugGameViewModel.kt

```diff
@@ -11,10 +12,11 @@ import com.communistutopia.spacetrader.model.Universe
  */
 
 class DebugGameViewModel : ViewModel() {
```

Comment:
just checking, we won't be using this class again right? Since we don't follow correct design implementation here, but I think it was just so show a list for ms5 or ms6.

Reply:
Yes we don't. Might be useful to show that our universe generation still works